Makefile

```diff
@@ -47,8 +47,13 @@ assets: $(REACT_APP_OUTPUT_DIR)
 	cd web/ui && GO111MODULE=$(GO111MODULE) GOOS= GOARCH= $(GO) generate -x -v $(GOOPTS)
 	@$(GOFMT) -w ./web/ui
 
+.PHONY: react-app-lint
+react-app-lint: 
```

Comment:
Could you add a `react-app-lint-fix` target as well that runs `yarn lint`? I think running it manually for now via `make react-app-lint-fix` would be fine, so other targets don't have to depend on it.

Reply:
Will add